.changeset/small-apples-brake.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sl-design-system/text-area": patch
```

Comment:
Or maybe `minor`? Because it's bigger change than just a fix? I mean it's going to be a breaking change :)